File: src/main/java/com/kbank/convenience/stock/stock/mediation/StockMediationApplication.java
package com.kbank.convenience.stock.stock.mediation;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.openfeign.EnableFeignClients;
import reactivefeign.spring.config.EnableReactiveFeignClients;

@SpringBootApplication
@EnableFeignClients
@EnableReactiveFeignClients
public class StockMediationApplication {

    public static void main(String[] args) {
        SpringApplication.run(StockMediationApplication.class, args);
    }

}

File: src/main/java/com/kbank/convenience/stock/stock/mediation/StockMediationApplication.kt
package com.kbank.convenience.stock.stock.mediation

import org.slf4j.LoggerFactory
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import reactivefeign.spring.config.EnableReactiveFeignClients

@SpringBootApplication
@EnableReactiveFeignClients
open class StockMediationApplication
inline fun <reified T> T.logger() = LoggerFactory.getLogger(T::class.java)!!
    fun main(args: Array<String>) {
        SpringApplication.run(StockMediationApplication::class.java, *args)
    }

